FT2023-340: Completed the Routing Module Task

RohanSingh-inno opened the pull request:
[1] Created a seperate folder which has two files.
[2] The first file is a .info.yml file which has the information regarding the custom module.
[3] The second file is a .module file which has the information about hooks.
[4] Implimented hook_cron, hook_form_alter and hook_help.

RohanSingh-inno committed:
FT2023-340: Understanding hooks completed.

RohanSingh-inno committed:
FT2023-340: Made Changes according to the mentor's review on peer's repo

RohanSingh-inno committed:
FT2023-340: Modified .gitignore file

[1]Moved the web/core/ to .gitignore

RohanSingh-inno committed:
FT2023-340: Verified and changed according to Code Sniffer.

RohanSingh-inno committed:
FT2023-340: Made Changes according to previous PR review.

vivekpan27 commented:
The PR title is incorrect.
The commit messages are incorrect.
Please fix them first.

RohanSingh-inno commented:
> 

I have pushed the code with update PR title and messages.

vivekpan27 commented:
I don't see ticket code in all commits.
For merge commits, ticket code not being is ok.
But for other commits it is required.
Please check how it can be fixed.

![image](https://github.com/RohanSingh-inno/Custom-Module/assets/25786248/c7d0720e-4ac9-46fc-8b1b-273c3c6e2ea6)

vivekpan27 commented on web/modules/custom/first_custom_module/first_custom_module.module:
```diff
@@ -0,0 +1,40 @@
+ *
+ * Used to alter the form structure value.
+ */
+function first_custom_module_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```
`array &$form`

RohanSingh-inno replied:
I have fixed this one.But I am not able to change the message of my initial commit.I have read about rebase and amend but  still facing issue doing this.I am getting merge conflict and when I am resolving this.It is giving fatal error.

RohanSingh-inno replied:
Sir I have completed the task.

vivekpan27 reviewed (commented):
Using rebase can you fix the previous commit titles.
Explore more on that front.

RohanSingh-inno committed:
FT2023-340: Hook Module Task Completed.

[1] Created a seperate folder which will hold the information regarding hook_module.

vivekpan27 commented:
1/ First fix the conflicts in the PR.
2/ Talk to a senior in office for help with rebase for fixing commit titles.

RohanSingh-inno commented:
> Add more commits by pushing to the `[FT2023-340](/RohanSingh-inno/Custom-Module/tree/FT2023-340)` branch on [RohanSingh-inno/Custom-Module](/RohanSingh-inno/Custom-Module).
> 
> [ Resolve conflicts ](/RohanSingh-inno/Custom-Module/pull/5/conflicts)

RohanSingh-inno committed:
FT2023-340: Final Code Pushed.

RohanSingh-inno committed:
FT2023-340: Final Commit.

RohanSingh-inno committed:
FT2023-340: Checking pushed Files.

RohanSingh-inno committed:
FT2023-340: Pushed Hook Module.
